ArticleTemplates/assets/scss/modules/_live-promo.scss

```diff
@@ -67,4 +68,31 @@
             display: block;
         }
     }
+
+    &__close-button {
+        border-radius: 1000px;
+        border: 1px solid #12121288;
+        position: absolute;
+        top: 12px;
+        right: 0;
+        width: 32px;
+        height: 32px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        padding: 0;
+
+        & div {
+            width: 14px;
+            height: 14px;
+
+            & svg {
+                display: block;
```

Comment:
`svg` are supposed to be "block" by default, but this actually make the layout work, otherwise it's slightly misaligned. So… not sure. Might be a peculiarity of iOS browser. We can adjust this if we need to when implementing the Android version.